### DISCUSSION TITLE
Loading snowboard.extras.css

### DISCUSSION THREAD

**Discussion by wverhoogt** [wintercms/winter#514] [Q&A] [answered]
I am developing using Snowboard and notice that snowboard.extras.css is not loaded if WinterCMS is installed in a subdirectory of the webroot. StylesheetLoader.js seems to assume that the css is reachable from the webroot. It contains the code:
```
if (!stylesLoaded) {
            const stylesheet = document.createElement('link');
            stylesheet.setAttribute('rel', 'stylesheet');
            stylesheet.setAttribute('href', '/modules/system/assets/css/snowboard.extras.css');
            document.head.appendChild(stylesheet);
        }
```
where this assumption is made.
I like to use subdirectories under the webroot to separate my WinterCMS projects.

**Comment by bennothommo** [comment, marked as answer]
@wverhoogt if you load in the `snowboard.extras.css` file manually in your template, then that StylesheetLoader will see that it's already loaded and won't try to inject it.

I think the reason I had to do this was because there's no way for us to know the base URL for the site directly in JavaScript (we'd have to pass it through somehow from the PHP side), so this was a safe assumption, but I'll make a note of it as an issue to look at.

**Reply by wverhoogt** [reply]
Ah, thanks for this suggestion. I assumed you would have to insert the real basePath from the PHP side, perhaps in the expansion of {% snowboard %}

**Reply by bennothommo** [reply]
@wverhoogt I've added base URL functionality in Snowboard, and adjusted the Stylesheet loader to use the correct base URL. See https://github.com/wintercms/winter/commit/192b91af123bc7fdb14045b37534c0027c9ea0e5

This will be released with the next Winter version.
